### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by bigspider**
Some improvements on the README to make it easier to follow. Mostly, revamping the "Running TEOS" section to make sure that all the steps are explicit and in the order they need to be executed by someone starting from scratch. Also fixed a couple of typos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,33 +43,34 @@ and / or
 
 - Add some global options when running the daemon (run `teosd.py -h` for more info).
 
-## Running TEOS in another network
+### Modifying the configuration file to run on regtest
```
[the remark this answers is not in the document]
Comment (sr-gi) [reply README.md:46]: See https://github.com/talaia-labs/python-teos/issues/170

File: DEPENDENCIES.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ Bitcoin needs to be running with the following options enables:
 - `txindex` to be able to look for non-wallet transactions
 - `server` to run rpc commands
 
-Here's an example of a `bitcoin.conf` you can use:
+Here's an example of a `bitcoin.conf` you can use if you want to run bitcoind on regtest:
```
[the remark this answers is not in the document]
Comment (bigspider) [reply DEPENDENCIES.md]: Added this example for mainnet in 3f014035ff43e022083b0a206b232088d8f5618e, while still leaving the example for regtest (which I modified to follow the same order of sections)

File: README.md [hunk 3]
```diff
@@ -24,21 +24,17 @@ Refer to [INSTALL.md](INSTALL.md)
 
 ## Running TEOS
 
-You can run `teos` buy running `teosd.py` under `teos`:
+Make sure bitcoind is running before running TEOS (it will fail at startup if it cannot connect to bitcoind). You can find
+[DEPENDENCIES.md#installing-bitcoind](here) a sample config file to run on regtest.
```
[the remark this answers is not in the document]
Comment (bigspider) [reply README.md]: Done in 6f89a84d43d376e97998c079fc6d40acceb9f8f4.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bigspider) [reply README.md]: Fixed in b58a46933f5462f75c8254bfd886b032a3ac7011 and f5ca514.

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (bigspider) [reply README.md]: Oops, I inverted link and description. Fixed in f5ca51488697cdfc3ec979b1acebe536203fabcc.

File: DEPENDENCIES.md [hunk 4]
```diff
@@ -61,28 +61,53 @@ Bitcoin needs to be running with the following options enables:
...
 txindex=1
 
-# [ZMQ]
+# [others]
+daemon=1
+debug=1
+maxtxfee=1
+```
+
+And here's an example for regtest:
```
Comment (sr-gi) [inline DEPENDENCIES.md:88]: Remove the regtest example. It's mainly the same just adding regtest=1. The mainnet one should be enough

File: README.md [hunk 5]
```diff
@@ -47,40 +43,48 @@ and / or
...
-By default, `teos` runs on `mainnet`. In order to run it on another network you need to change the network parameter in the configuration file or pass the network parameter as a command line option. Notice that if teos does not find a `bitcoind` node running in the same network that it is set to run, it will refuse to run.
+### Tower identity and signing key
 
+In order to run `teos`, you need to have a pair of keys stored in your data directory, that are used as the tower's identity. Users can recognize the tower by its public key, and the tower uses the private key to sign appointments. You can follow [generate keys](#generate-keys) to generate them.
 
-### Modifying the configuration file
+### Starting the TEOS daemon 👁
 
-The configuration file option to change the network where `teos` will run is `btc_network` under the `bitcoind` section:
+Not that everything is ready, you can run `teos` by running `teosd.py` under `teos`:
```
Comment (sr-gi) [inline README.md:52]: Typo: Not -> Now

File: README.md [hunk 6]
```diff
@@ -47,40 +43,48 @@ and / or
 
 - Add some global options when running the daemon (run `teosd.py -h` for more info).
 
-## Running TEOS in another network
-
-By default, `teos` runs on `mainnet`. In order to run it on another network you need to change the network parameter in the configuration file or pass the network parameter as a command line option. Notice that if teos does not find a `bitcoind` node running in the same network that it is set to run, it will refuse to run.
+### Tower identity and signing key
```
Comment (sr-gi) [inline README.md:46]: Let's do Tower id and signing key

File: README.md [hunk 7]
```diff
@@ -47,40 +43,48 @@ and / or
...
 
+In order to run `teos`, you need to have a pair of keys stored in your data directory, that are used as the tower's identity. Users can recognize the tower by its public key, and the tower uses the private key to sign appointments. You can follow [generate keys](#generate-keys) to generate them.
```
Comment (sr-gi) [inline README.md:48]: I'd change it for:

`teos` needs a pair of keys that will serve as tower id and signing key. The former can be used by users to identify the tower, whereas the latter is used by the tower to sign responses. You can follow [generate keys](#generate-keys) to generate them.

File: DEPENDENCIES.md (hunk 4 above)
Comment (bigspider) [reply DEPENDENCIES.md]: Done in ada9645.

File: README.md (hunk 5 above)
Comment (bigspider) [reply README.md]: Done in ada9645.

File: README.md (hunk 7 above)
Comment (bigspider) [reply README.md]: Done in ada9645.

File: README.md (hunk 6 above)
Comment (bigspider) [reply README.md]: Done in ada9645.

**Pull request merged**